### DISCUSSION TITLE
Initiative timer

### DISCUSSION THREAD

**Discussion by JimRoepcke** [Lurkars/gloomhavensecretariat#528] [Ideas]
I have lots of time right now waiting for one of my party mates to decide on their initiative to ask for a feature to automatically start a timer for each character when a new round starts. The timer stops once their initiative is entered. Ideally somewhere the app shows (for each character) the total / max / min time to enter the initiative, perhaps in the screen accessible by tapping on the character's name.

**Comment by Lurkars** [comment]
I may think about this. It's sure a nice to have, but still very specific use case.
